inst/extdata/install-packages.R

```diff
@@ -87,6 +87,11 @@ get_PKSim_Minimal <- function(install_option) {
     # test if PK-Sim.R dll can be loaded
     ospsuite::initPKSim(pksimFolderPath = "PKSim")
 
+    cat('ospsuite::initPKSim(pksimFolderPath = "PKSim")',
```

Comment:
What was the reason for this, again?

I need more convicing, I don't like the idea of loading some libraries at project startup. This would also prevent ospsuite package from updating, because it cannot update the dlls once loaded.

Reply:
You are right, looked convenient but this will confuse users (I'm used to commenting/uncomment Rprofile files but certainly not typical users.) I think we can close